traveller/modules/cfp/view.py

```diff
@@ -46,6 +47,7 @@ def add_talk(year):
         talk.author_list = AuthorList()
     talk.author_list.authors.append(current_user)
     talk.submitter_id = current_user.id
+    talk.year_submitted = str(dt.utcnow().year)
```

Comment:
Change to = year as we are already passing year

Reply:
Sure